tests/performance-test/performance-test.sh

```diff
@@ -26,17 +24,62 @@ NOTES:
   * telemetry-bench is recommended since there are problems getting collectd-tg to scale concurrently
   * telemetry-bench somewhat underperforms (runs too slow), but every message does get sent
   * A plugin setting of 1000 reasonably matches the plugins/host we expect to see from OSP
-
 EXAMPLES:
   Quick minimal test ~1k/s (1 min)
   ./performance-test.sh -t tb -c 60 -h 1 -p 1000 -i 1 -n 1
-
   Recommended command for ~20k/s (10 mins)
   ./performance-test.sh -t tb -c 600 -h 5 -p 1000 -i 1 -n 4
 ENDUSAGE
     exit 1
 }
 
+delete(){
+   oc delete servicemonitor/prometheus \
```

Comment:
:-1: 
I'm not sure I like the idea of deleting the service monitors and waiting for them to re-establish each time we launch some tests. Currently, once my environment is set up, I can execute this script and the test starts up relatively quickly. Now I need to wait for the Prometheus operator to deal with the service monitors each time, and I know from experience that's slow. I run it a few times an hour these days when I'm working on it actively, so it will add up.

performance-test-tb.sh already has `oc delete job/saf-performance-test` in it, so it seems superfluous here. For the service monitors, I think if you change `oc create` to `oc apply` on L50 and L51 below, then you'll find we don't need to delete anything here. Using "apply" is not always safe compared to a delete/create, but I think for our purposes it will be totally fine.